src/inputParser.js

```diff
@@ -0,0 +1,47 @@
+import { isValidSeparator } from "./validate.js";
+
+const inputParser = {
+
+    getSeparator({ inputText }) {
+        if(inputText.startsWith("//")) {
+            //커스텀 구분자 입력으로 시작할 경우 커스텀 구분자를 구하는 함수를 사용한다.
+            return this.setSeparator({ inputText })
+        } else {
+            return [inputText, null];
+        }
```

Comment:
앞에 if문에서 return을 했기 때문에, 뒤에 else 없이도 바로 return해도 좋을 것 같습니다:) [16.3](https://github.com/ParkSB/javascript-style-guide#blocks--no-else-return)
```suggestion
        if(inputText.startsWith("//")) {
            //커스텀 구분자 입력으로 시작할 경우 커스텀 구분자를 구하는 함수를 사용한다.
            return this.setSeparator({ inputText })
        }
        
        return [inputText, null];
```